lib/index.js

```diff
@@ -12,6 +13,13 @@ const bundle = ({ file, options, source, files }) => {
 
   const readable = new Readable();
   const filePath = path.join(source, file);
+
+  if (!fs.existsSync(filePath)) {
+    return Promise.reject(
+      new Error(`Cannot use '${file}' as an entry point - file does not exist.`)
```

Comment:
Maybe `Can't find '${file}'. The file doesn't exist or the path is invalid.` would be a bit more descriptive here.